src/gconf-io-p.h

```diff
@@ -27,125 +27,142 @@
-   CHKERR (rc,err_ret,dir);                                            \
-}
+#define CHKERR(rc, err_ret, dir)                                  \
+	{                                                             \
+		if (FALSE == rc)                                          \
+		{                                                         \
+			printf ("GTT: GConf: Warning: set %s failed: ", dir); \
+			if (err_ret)                                          \
+				printf ("%s", err_ret->message);                  \
+			printf ("\n");                                        \
+		}                                                         \
+	}
+
+#define SETBOOL(dir, val)                                                  \
+	{                                                                      \
+		gboolean rc;                                                       \
+		GError *err_ret = NULL;                                            \
+                                                                           \
+		rc = gconf_client_set_bool (client, GTT_GCONF dir, val, &err_ret); \
+		CHKERR (rc, err_ret, dir);                                         \
+	}
+
+#define F_SETINT(dir, val)                                      \
+	{                                                           \
+		gboolean rc;                                            \
+		GError *err_ret = NULL;                                 \
+                                                                \
+		rc = gconf_client_set_int (client, dir, val, &err_ret); \
+		CHKERR (rc, err_ret, dir);                              \
+	}
+
+#define SETINT(dir, val) F_SETINT (GTT_GCONF dir, val)
+
+#define F_SETSTR(dir, val)                                          \
+	{                                                               \
+		gboolean rc;                                                \
+		GError *err_ret = NULL;                                     \
+                                                                    \
+		const gchar *sval = val;                                    \
+		if (!sval)                                                  \
+			sval = "";                                              \
+		rc = gconf_client_set_string (client, dir, sval, &err_ret); \
+		CHKERR (rc, err_ret, dir);                                  \
+	}
+
+#define SETSTR(dir, val) F_SETSTR (GTT_GCONF dir, val)
+
+#define SETLIST(dir, tipe, val)                                                  \
+	{                                                                            \
+		gboolean rc;                                                             \
+		GError *err_ret = NULL;                                                  \
+                                                                                 \
+		rc = gconf_client_set_list (client, GTT_GCONF dir, tipe, val, &err_ret); \
+		CHKERR (rc, err_ret, dir);                                               \
+	}
```

Comment:
Where are you on the whole tabs-versus-space debacle?

This project seems to have used a lot of tabs originally - on the other hand, GTK, Gnome, LLVM all seem to advocate for always spaces for indenting. (Although I find their indent width of 2 a bit challenging, much preferring 4 myself.)

I'm a bit ambivalent in this case - personally I do prefer space, but this code did use tabs originally. I'm not sure if your previous PR on formatting using space was because of preference or accident, but I guess if you like space-indenting too, we might as well apply it. It would resolve alignment issue like this. And regarding diff size, I don't think we should worry too much about changes in leading whitespace, since that is usually easy to ignore in diff tools.